automatic/apache-httpd/tools/helpers.ps1

```diff
@@ -4,7 +4,7 @@ function Assert-TcpPortIsOpen {
         [Parameter(Position = 0, Mandatory, ValueFromPipeline)][ValidateNotNullOrEmpty()][int] $portNumber
     )
 
-    $process = ( netstat -aon | foreach { if ( $_ -match ":$portNumber" ) { $_ } } )-split " " | `
+    $process = (( netstat -aon ) -match ":$portNumber" ) -split " " | `
```

Comment:
match is not good, please fix regex. This one will return invalid result, for example you searching for `:80` may return `:8080`.

Reply:
fixed the match